### PR TITLE
Create description pages for thing samples apps from zephyr-knot-sdk

### LINKED ISSUE #32: Create description pages for thing samples apps

### PR DESCRIPTION
**Pull request by alla3**
Create description pages for the following samples apps:

- hello
- multisensor
- plate
- tank
- thermo
- toggle

Closes #32

### CODE REVIEW

File: doc/thing/samples/basic-samples/toggle.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+toggle
+======
+
+Overview
+--------
+
+The toggle example uses an on-board LED to toggle its state via cloud
+request.
```
Comment (jcorde) [inline doc/thing/samples/basic-samples/toggle.rst]: Please, describe which LED is controlled via cloud request.

File: doc/thing/samples/basic-samples/thermo.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
+thermo
+======
+
+Overview
+--------
+
+The thermo example simulates a device showing the room temperature.
+The purpose of this sample is to show how to register an integer value
+as a sensor using the KNoT Zephyr SDK.
+
```
Comment (jcorde) [inline doc/thing/samples/basic-samples/thermo.rst]: Please, make it clear that the value is automatically updated. And how it is updated

File: doc/thing/samples/basic-samples/tank.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
+tank
+====
+
+Overview
+--------
+
+The tank example simulates a device showing the current tank volume.
+The purpose of this sample is to show how to register a floating-point
+value as a sensor using the KNoT Zephyr SDK.
+
```
Comment (jcorde) [inline doc/thing/samples/basic-samples/tank.rst]: Please, make it clear that the value is automatically updated. And how it is updated

File: doc/thing/samples/basic-samples/plate.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
+plate
+=====
+
+Overview
+--------
+
+The plate example simulates a device registering the read car plate in a
+sensor. The purpose of this sample is to show how to register a string value
+as a sensor using the KNoT Zephyr SDK.
+
```
Comment (jcorde) [inline doc/thing/samples/basic-samples/plate.rst]: Please, make it clear that the value is automatically updated. And how it is updated

File: doc/thing/samples/basic-samples/toggle.rst (hunk 1 above)
Comment (alla3) [reply doc/thing/samples/basic-samples/toggle.rst]: Ok.

File: doc/thing/samples/basic-samples/thermo.rst (hunk 2 above)
Comment (alla3) [reply doc/thing/samples/basic-samples/thermo.rst]: Ok.

File: doc/thing/samples/basic-samples/tank.rst (hunk 3 above)
Comment (alla3) [reply doc/thing/samples/basic-samples/tank.rst]: Ok.

File: doc/thing/samples/basic-samples/plate.rst (hunk 4 above)
Comment (alla3) [reply doc/thing/samples/basic-samples/plate.rst]: Ok.

**Pull request merged**